mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.Opus.Plugin.Mixfile do
   use Mix.Project
 
-  @version "0.16.0"
+  @version "0.16.1"
```

Comment:
If docs/readme are the only change, then there's no need to bump the version. Docs can be published with `mix hex.publish docs`